### ISSUE TITLE
Add SourceLink support

### ISSUE THREAD

**Issue by wiverson** [supabase-community/supabase-csharp#85]
# Feature request

Add https://github.com/dotnet/sourcelink support to allow for easier source debugging.

**Comment by acupofjose** [comment]
Looks like a great idea to me!

**Comment by wiverson** [comment]
FYI, I have a bunch of bug fixes incoming shortly. Figured it was good to get to where the app was solid w/Unity on iOS and a bunch of the dumb little stuff (e.g. minor threading/file system stuff) was all golden. I decided to just focus for a few weeks on finishing up a test version of the app, it's coming along very nicely. I'm guessing those bug fixes will be incoming over the next 1-2 weeks.

I'll look at the tweaks for this sourcelink stuff after. This was mostly a reminder lol
